scaffold/template/custom/.crowdbotics.json

```diff
@@ -1,7 +1,7 @@
 {
   "scaffold": {
     "type": "react-native",
-    "version": "2.0.0",
+    "version": "2.1.0",
```

Comment:
Will we do this change everytime we do change in modules scaffold now?

Reply:
Yes, there is a GitHub Action that checks for this now so if the version hasn't been updated it will fail. We need to do this every time the scaffold changes and also update the config to store which versions can be upgraded.

Reply:
Have you merged from master? The most recent version is 2.1.1 not 2.0.0 @mabdullahabid

Reply:
Yeah I am rebasing as we speak! The version needs to be bumped up now cause there was a merge to master.